src/client/options/mod.rs

```diff
@@ -250,6 +250,12 @@ impl ServerAddress {
         }
     }
 
+    pub(crate) fn into_host(self) -> String {
+        match self {
+            Self::Tcp { host, .. } => host,
```

Comment:
A method like this could lead to accidentally discarding the port, so I think we'll want to omit it.

Reply:
Done.